### PR TITLE
Feat/toast

### PR DESCRIPTION
**Pull request by abhij1607**
This PR adds toast feature
Now user can see toast on : 

- video like/dislike
- playlist add/delete
- video added/deleted in playlist
- video added/deleted in watch later

### CODE REVIEW

**Review by adith-a-danthi — COMMENTED** [review]
Works as intended. Nice touch adding the toast for error conditions as well. Good to merge

File: src/utils/server-request.js [hunk 1]
```diff
@@ -99,7 +106,9 @@ const requestAddVideoInPlaylist = async (
       type: "UPDATE_SINGLE_PLAYLIST",
       payload: response.data.playlist,
     });
+    notify("Added Video in Playlist");
   } catch (error) {
+    notify("Added Video in Playlist");
     console.log(error);
```
Comment (stutisk) [inline src/utils/server-request.js:112]: console.log can be removed

**Review by stutisk — APPROVED** [review]

**Review by stutisk — APPROVED** [review]
code looks good 
good to merge

File: src/utils/server-request.js (hunk 1 above)
Comment (abhij1607) [reply src/utils/server-request.js:112]: thanks for suggestion

**Pull request merged**